test/src/test/java/hudson/triggers/TriggerTest.java

```diff
@@ -0,0 +1,90 @@
+package hudson.triggers;
+
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016 Felix Belzunce Arcos
```

Comment:
NIT: © should be the first thing in the file (it should come before the package)